src/pdfflow/configflow.py

```diff
@@ -3,23 +3,24 @@
 """
 # Most of this can be moved to a yaml file without loss of generality
 import os
-
+import logging
 import numpy as np
+import tensorflow as tf
```

Comment:
Move the tensorflow below the `os.environ.setdefault`, otherwise the enviroment variables don't get applied.